### ISSUE TITLE
Process required: `h4` on bookings

### ISSUE THREAD

**Issue by DaveyJH** [DaveyJH/ci-portfolio-four#98]
**Process:**
Replace `h4` element with `h3` element on bookings page

**Benefit:**
[wave accessibility checker](https://wave.webaim.org/report#/https://mindful-massage.herokuapp.com/bookings/book/) reports an alert with a skipped heading. This should be resolved for better HTML semantics

---
*Notes:*

**Comment by DaveyJH** [comment]
2736484 does **NOT** fix this, that is a typo
